packages/field-plugin/src/createFieldPlugin/createFieldPlugin.ts

```diff
@@ -9,7 +9,10 @@ import { isCloneable } from '../utils/isCloneable'
 
 export type CreateFieldPluginOptions<Content> = {
   onUpdateState: (state: FieldPluginResponse<Content>) => void
-  parseContent: (content: unknown) => Content
+  validateContent?: (content: unknown) => {
```

Comment:
maybe ValidateContent can be a separate type that we can reuse

Reply:
I actually tried it but somehow it broke the types. and... I tried it again and it's working. I must've done it wrongly before 😂
https://github.com/storyblok/field-plugin/pull/280/commits/d5ffd05f6ab838d7de6d9fa220f307f09ea37d3a